instance/api/instance.py

```diff
@@ -47,7 +47,9 @@ def provision(self, request, pk=None):
         Start the (re-)provisioning of an instance
         """
         instance = self.get_object()
-        if instance.status not in (instance.EMPTY, instance.READY):
+        if instance.status not in (instance.EMPTY,
+                                   instance.READY,
+                                   instance.PROVISIONING_FAILURE):
```

Comment:
Is there a way to improve this? To not have to maintain this list when we add new states, and avoid duplicating the test in other places - like the client for example.